devops/tartarus-helm/values.yaml

```
@@ -0,0 +1,81 @@
+    DB_NAME: ""  # The name of the Postgres database. Default is "datastore".
+    DB_USER: ""  # The Postgres username. Default is "postgres".
+    DB_PASSWORD: ""  # The Postgres password. Set this securely.
+
+global:
+  postgresql:
+    enabled: true  # Whether the Postgres database should be enabled as part of this deployment.
+    
+    auth:
+      # The default username for the Postgres database. Modify this if you're using a custom username.
+      username: "postgres"
+      
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)